rpm/qtbase.spec

```diff
@@ -543,6 +543,10 @@ install -D -p -m 0644 %{_sourcedir}/macros.qt5-default \
 mkdir -p %{buildroot}/etc/xdg/qtchooser
 ln -s %{_sysconfdir}/xdg/qtchooser/5.conf %{buildroot}%{_sysconfdir}/xdg/qtchooser/default.conf
 
+# Add Qt binaries to PATH
+mkdir -p %{buildroot}/%{_sysconfdir}/profile.d/
+echo "export PATH=$PATH:"%{_libdir}/qt5/bin > %{buildroot}/%{_sysconfdir}/profile.d/qt_bin_path.sh
```

Comment:
Built this locally and now i have a file with content "export PATH=/home/pvuorela/src/mer-qt/qtbase/.mb2/wrappers:/home/pvuorela/src/mer-qt/qtbase/.mb2/user-wrappers:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/pvuorela/.local/bin:/home/pvuorela/bin:/home/pvuorela/.local/bin:/home/pvuorela/bin:/usr/lib/qt5/bin" 

Looks like it's expanding the $PATH when doing the eho.

Reply:
Fixed.